.gitignore

```diff
@@ -83,3 +82,10 @@ test.tap
 # Xcode workspaces and project folders
 *.xcodeproj
 *.xcworkspace
+
+# mac installer files
+/tools/osx-pkg/osx-pkg-out.pkgproj
+/tools/osx-pkg/strings/LICENSE.txt
+/tools/osx-pkg/strings/**/*.out.rtf
+/tools/osx-pkg/scripts/iojs-create-node-symlink
+/tools/osx-pkg/scripts/iojs-run-uninstall
```

Comment:
You can get rid of L90 `/tools/osx-pkg/scripts/iojs-create-node-symlink`, and then change L91 to `/tools/osx-pkg/scripts/nodejs-run-uninstall`